docs/docs_project_history.md

```diff
@@ -0,0 +1,94 @@
+# Project History
+
+November 21, 2015 - the decision to start development.
+
+### SkyMP 0.0.1 Pre Alpha
+
+Written in C++. Only basic Movement sync. Didn't have own Git repo at the moment of creation.
+
+First tested September 17, 2016.
+
+* [plugin_example (skymp dll)](https://github.com/Pospelove/plugin_example-skymp-dll)
+
+### SkyMP ???
+
+Some versions were lost. Didn't use version control systems.
+
+### SkyMP 0.9, 1.0+
+
+Written in C++. Insipired by GTA SA-MP and TES3MP. March 2017 - June 2018.
+
+First tested in September 2017.
+
+Has Lua scripting system.
+
+* [SkyMP Client](https://bitbucket.org/skymp_team/skymp-client)
+* [SkyMP Dedicated Server](https://bitbucket.org/skymp_team/skymp-dedicated-server)
+* [SkyMP Official GameMode](https://bitbucket.org/skymp_team/skymp-official-gamemode) - Lua gamemode for SkyMP called Secunda.
+* [Reman](https://bitbucket.org/skymp_team/reman) - Tool for remote server launching written in Python.
+* [Secunda World](https://bitbucket.org/skymp_team/secunda-world) - World backup from OBT 2018.
+
+### SkyMP 2.0+
+
+Written in C++ and JavaScript. June 2018 - March 2019.
+
+Stateless server storing everything in MongoDB.
+
+Has client-side Lua scripting system.
+
+Most popular version at the moment of publishing this: 30k+ downloads on Open Beta 2019 launch, 1000/1000 players online. We decided to stop Open Beta due to technical problems.
+
+* [skymp2](https://gitlab.com/pospelov/skymp2)
+* [skymp2-installer](https://gitlab.com/pospelov/skymp2-installer)
+* [secunda2](https://gitlab.com/pospelov/secunda2)
```

Comment:
Would be great if you have any memories left about difference against first version of Secunda